index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Cluedo Reminisce</title>
+</head>
+
+<body>
+    <h1> CL<span class="red">U</span>EDO </h1>
+    <h3>The classic <span class="yellow">mystery</span> game</h3>
+    <section class="cards">
+      <article class="killer-deck" id="killerCard">
+        <p class="title-font">KILLER</p>
+        <img id="killerImage" src="" height="200" />
+        <p class="second-title-font" id="killerName"></p>
+        <p class="third-title-font" id="killerAge"></p>
+        <p class="third-title-font" id="killerOccupation"></p>
+        <p class="third-title-font" id="killerDescription"></p>
+        <p class="third-title-font" id="killerFavouriteWeapon"></p>
+      </article>
+      <article class="weapon-deck" id="weaponCard">
+        <p class="title-font">WEAPON</p>
+        <img class="weapon-image" id="weaponImage" src="" height="130" />
```

Comment:
This height-attribute is inline-css and could be moved to the css-file.

Reply:
I've moved this now and also with the killer deck card image :)